pkg/reconciler/jobsource/controller.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jobsource
+
+import (
+	"context"
+
+	jsclient "github.com/n3wscott/sources/pkg/client/injection/client"
```

Comment:
this is suppose to be the `sourcesclient` because it will be the client for all sources, not just jobsource.